modules/core/client/directives/pagination.client.directive.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+//Pagination Directive
+angular.module('core').directive('meanPagination', function () {
+    return {
+        restrict: 'AEC',
+        scope: {
+            meanPaginationData: '=meanPaginationData',
```

Comment:
meanPaginationUrl has to be an @ as it needs to be interpreted as a string.  Same with Search since that is what I am testing ng-show with. I could possibly change the Search to an =.